org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/TypePattern.java

```diff
@@ -187,10 +187,10 @@ final ASTNode internalGetSetChildProperty(ChildPropertyDescriptor property, bool
 	 * </ul>
 	 * @exception UnsupportedOperationException if this operation is used other than JLS19
 	 * @exception UnsupportedOperationException if this expression is used with previewEnabled flag as false
-	 * @Deprecated(forRemoval = true, since="2024-07")
 	 * In the JLS22 API, this method is replaced by  <code>setPatternVariable(VariableDeclaration patternVariable)</code>
```

Comment:
```suggestion
	 * @deprecated In the JLS22 API, this method is replaced by  <code>setPatternVariable(VariableDeclaration patternVariable)</code>
```

Reply:
@laeubi could you please mention, what are the things should I modify in this PR.  The code you have mentioned here has changed. The `@deprecated` no longer using in the javaDoc in our context.

Reply:
You need both:

1. `@deprecated` **javadoc tag** mentions what should used instead in a descriptive way for the user
2. `@Deprecated` **annotation** tells the compiler that this method is deprecated (maybe for removal) to emit warnings/error to the user while compilation.

Reply:
Thanks you so much for the details. I have pushed the latest changes, could you please review the changes.